### DISCUSSION TITLE
Date and time stopped showing in bar

### DISCUSSION THREAD

**Discussion by oliveilment** [i3/i3#4888] [Q&A]
Using i3 with void linux. Suddenly tztime module stopped in i3status showing a blank space in the bar. No error messages though. All other default modules are working. Im using the default /etc/i3status.conf file. i3status version 2.14 and i3 version 4.20. Tried i3status.conf file from https://github.com/i3/i3status/tree/master/etc also.

**Comment by Airblader** [comment]
What if you run i3status from a terminal, do you observe the same?

**Reply by oliveilment** [reply]
In terminal date and time displayed correctly with all other modules.

**Reply by Airblader** [reply]
My guess would then be that there's an issue with our locale setup (in the environment in which i3status runs).
